src/helpers/formValidations.js

```diff
@@ -0,0 +1,7 @@
+export function onlyAllowNonZeroDigits(event) {
```

Comment:
Is this the right place for these types of helpers?  I figured we might use this somewhere else down the line, so it made sense to isolate from `TeamForm.js`.